kiosk/src/BackendRequests.ts

```diff
@@ -10,15 +10,13 @@ export const getGameCodeAsync = async (kioskCode: string) => {
         e.name = "PollError";
         throw e;
     } else {
-        const gameCodeList = JSON.parse((await response.json())?.shareIds);
-        if (gameCodeList?.length) {
-            return gameCodeList;
-        }
+        const gameCodeEntries = JSON.parse((await response.json())?.shareIds);
+        return gameCodeEntries;
```

Comment:
Should we update the function name to `getGameCodesAsync` to reflect that we are getting one or more?